src/filesystem/impls/filer/ArchiveUtils.js

```diff
@@ -281,35 +280,37 @@ define(function (require, exports, module) {
             path = Path.resolve(root, path);
             var basedir = Path.dirname(path);
 
-            if (_skipFile(path)) {
+            if(_skipFile(path)) {
                 return callback();
             }
 
-            fs.mkdirp(basedir, function (err) {
-                if (err) {
+            fs.mkdirp(basedir, function(err) {
+                if(err) {
```

Comment:
You're right @humphd. I'm going to put this code into a function and test it out.